bucket/janet.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": "1.13.1",
+    "description": "Lisp-like, embeddable, functional and imperative programming language and bytecode interpreter",
+    "homepage": "https://janet-lang.org",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/janet-lang/janet/releases/download/v1.13.1/janet-v1.13.1-windows-x64-installer.msi",
+            "hash": "ef5d32257594c8ab54aaa52fe8e84bb951a4883bbb6f6848e845759122d33a35"
+        },
+        "32bit": {
+            "url": "https://github.com/janet-lang/janet/releases/download/v1.13.1/janet-v1.13.1-windows-x86-installer.msi",
+            "hash": "627a0dccef7815fdde735caf3ea2eb3e16f2ec26ac3d0cc365cf8ea67589d5e3"
+        }
+    },
+    "extract_dir": "Janet",
+    "bin": [
+        "bin\\janet.exe",
+        "bin\\jpm.bat"
```

Comment:
https://janet-lang.org/docs/index.html

> If you want to use jpm to install native extensions, you will also need to install Visual Studio, ideally a recent release (2019 or 2017 should work well). Then, running jpm from the x64 Native Tools Command Prompt should work for a 64-bit build, and from the Developer Command Prompt should work for 32-bit builds. Using these specific command prompts for jpm simply lets jpm use the Microsoft compiler to compile native modules on install.

Reply:
We have two (decent) options I guess. 
1. Put a note in notes section about `jpm` depending on Visual C
2. Remove jpm.bat shim

Reply:
@bakpakin What do you suggest?